File: app/src/main/java/vn/loitp/app/activity/customviews/seekbar/verticalseekbar/VerticalSeekbarActivity.java
package vn.loitp.app.activity.customviews.seekbar.verticalseekbar;

import android.os.Bundle;
import android.view.View;
import android.widget.SeekBar;

import loitp.basemaster.R;
import vn.loitp.core.base.BaseFontActivity;
import vn.loitp.core.utilities.LLog;
import vn.loitp.views.seekbar.verticalseekbar.VerticalSeekBar;

//https://github.com/h6ah4i/android-verticalseekbar
public class VerticalSeekbarActivity extends BaseFontActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        final VerticalSeekBar seekBar1 = (VerticalSeekBar) findViewById(R.id.seekBar1);
        seekBar1.setMax(100);
        seekBar1.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
            @Override
            public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
                LLog.d(TAG, "onProgressChanged " + progress);
            }

            @Override
            public void onStartTrackingTouch(SeekBar seekBar) {
                LLog.d(TAG, "onStartTrackingTouch");
            }

            @Override
            public void onStopTrackingTouch(SeekBar seekBar) {
                LLog.d(TAG, "onStopTrackingTouch");
            }
        });
        findViewById(R.id.bt).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                seekBar1.setProgress(30);
            }
        });
    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_vertical_seekbar;
    }
}

File: app/src/main/java/vn/loitp/app/activity/customviews/seekbar/verticalseekbar/VerticalSeekbarActivity.kt
package vn.loitp.app.activity.customviews.seekbar.verticalseekbar

import android.os.Bundle
import android.widget.SeekBar
import com.annotation.IsFullScreen
import com.annotation.LogTag
import com.core.base.BaseFontActivity
import kotlinx.android.synthetic.main.activity_seekbar_vertical.*
import vn.loitp.app.R

//https://github.com/h6ah4i/android-verticalseekbar

@LogTag("VerticalSeekbarActivity")
@IsFullScreen(false)
class VerticalSeekbarActivity : BaseFontActivity() {

    override fun setLayoutResourceId(): Int {
        return R.layout.activity_seekbar_vertical
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        seekBar1.max = 100
        seekBar1.setOnSeekBarChangeListener(object : SeekBar.OnSeekBarChangeListener {
            override fun onProgressChanged(seekBar: SeekBar, progress: Int, fromUser: Boolean) {
                logD("onProgressChanged $progress")
            }

            override fun onStartTrackingTouch(seekBar: SeekBar) {
                logD("onStartTrackingTouch")
            }

            override fun onStopTrackingTouch(seekBar: SeekBar) {
                logD("onStopTrackingTouch")
            }
        })
        btSetProgress.setOnClickListener { _ -> seekBar1.progress = 30 }
    }

}
